specification/sql/resource-manager/readme.md

```diff
@@ -78,6 +78,7 @@ input-file:
 - Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json
 - Microsoft.Sql/preview/2017-10-01-preview/cancelOperations.json
 - Microsoft.Sql/preview/2017-10-01-preview/cancelPoolOperations.json
+- Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json

```

Comment:
Please add to both package-composite-v1 and -v2 sections, and package-pure-2017-10-preview